module6/lessons/Week5/NetworkingInTheJobHunt.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page
+title: Networking in the Job Hunt
+---
+
+## Learning Goals
+* Have a checklist of steps to follow when networking in the job hunt
+* Have a clear next step at any point to help avoid overwhelm/freezing
+
+## Intro
+
+You know that networking and the human connection part of hustling is important. You're convinced that it's a key tool in landing that first job. But how do you choose who to reach out to? And what do you say? When do you follow? This guide will give suggestions in all of those areas. As always, find what works best for you and keep trying things. But here is a place to start!
+
+Start a document and keep track of your companies, people, and who you've messaged there!
+
+## 1. Start with Target Companies
+
+Step one is to compile a list of target companies. 
+
+Criteria for a target company:
+1. Company that hires developers like you. The company should meet some of the following criteria:
+    * Hiring remotely or in locations you're interested in
+    * .NET/C#
+    * Has hired Turing grads in the past
+    * Open to hiring developers without CS degrees

```

Comment:
Do we want to add something about how this might be determined?  Like, look for employees on linked in with dev titles and bootcamps listed under experience? Or, is there an easier way that you know of 😆

Reply:
Good call the job description and looking at current employees on LinkedIn is how I would do this.